certified-connectors/Zellis/apiProperties.json

```diff
@@ -0,0 +1,300 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "scopes": [
+            "{Scope}"
+          ],
+          "redirectMode": "Global",
```

Comment:
Please update "redirectMode": "Global" to "redirectMode": "GlobalPerConnector"

Reply:
Have made the changes and update the PR

Reply:
The changes are not reflected.  Please check.

Reply:
@vmanoharas This file shows as outdated but not sure why change is not seen here which is present on REPO